src/modules/faucet/FaucetAssetsList.tsx

```diff
@@ -54,6 +58,20 @@ export default function FaucetAssetsList() {
     );
   }
 
+  if (currentMarketData.enabledFeatures?.external_faucet) {
+    return (
+      <ListWrapper titleComponent={''}>
+        <Box display="flex" justifyContent="center" alignItems="center" marginBottom="39px">
+          <Button color="primary" variant="contained" onClick={handleOpenExternalFaucet}>
```

Comment:
Since this is a link, you can set the component prop to be a `Link` component, and then set the href prop. Then you don't need the handler function. You can look at `BridgeButton.tsx` for a similar example.